### PR TITLE
Finish basic login

### PR DESCRIPTION
**Pull request by hoangnguyen293**
screenshot from 2018-07-11 11-09-27

### CODE REVIEW

**Comment by hoangnguyen293** [comment]
ready

**Comment by hoangnguyen293** [comment]
ready

File: app/views/sessions/new.html.erb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<% provide(:title, t(".log_in")) %>
+<h1>Log in</h1>
```
Comment (phamvanbk11) [inline app/views/sessions/new.html.erb]: I18n

File: app/helpers/sessions_helper.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  end
+
+  # Return true if the user is logged in, false otherwise
+  def logged_in?
+    !current_user.nil?
+  end
+
+  # Logs out the current user.
+  def log_out
+    session.delete(:user_id)
```
Comment (phamvanbk11) [inline app/helpers/sessions_helper.rb]: xóa ()

**Comment by phamvanbk11** [comment]
commented

**Comment by hoangnguyen293** [comment]
ready

**Comment by phamvanbk11** [comment]
merged

**Pull request merged**